ntfy/cli.py

```diff
@@ -32,6 +37,28 @@ def run_cmd(args):
         'failed', *map(int, divmod(duration, 60)))
 
 
+def watch_pid(args):
+    if psutil is None:
+        logging.error(
+            "This command requires psutil module. Pleases install psutil.")
+        exit(1)
+    try:
+        p = psutil.Process(args.pid)
+        cmd = p.cmdline()
+    except psutil.NoSuchProcess:
+        logging.error("PID {} not found".format(args.pid))
+        exit(1)
+    p.wait()
+    if emojize is not None and not args.no_emoji:
+        prefix = '\u2705 ' if retcode == 0 else '\u274C '
+    else:
+        prefix = ''
+    return '{}"Process[{}]: {}" was finished at {}'.format(
```

Comment:
Can you not find out the start time and compute run time? Most notifications include the date already :wink: